custom/icds_core/view_utils.py

```diff
@@ -9,7 +9,7 @@
 from corehq.apps.users.models import DomainMembershipError
 from django.http import HttpResponse
 
-DATA_INTERFACE_ACCESS_DENIED = mark_safe(ugettext_lazy(
+DATA_INTERFACE_ACCESS_DENIED = mark_safe(ugettext_lazy(  # nosec: no user input
```

Comment:
do you know why this needs to be marked safe?

Reply:
Nope. It has multiple problems -- the string itself shows no reason to be marked safe, the `mark_safe` immediately defeats the point of the lazy translation, and the code is almost certainly unused (because ICDS is gone). This was the one area where I just did the simplest thing possible because the code almost certainly did not matter, and I didn't want to deal with the complexity of deleting the file.

Reply:
ha, fair.